packages/frontend/editor-ui/src/components/canvas/experimental/experimentalNdv.store.ts

```diff
@@ -137,12 +142,14 @@ export const useExperimentalNdvStore = defineStore('experimentalNdv', () => {
 		previousZoom: computed(() => previousViewport.value),
 		collapsedNodes: computed(() => collapsedNodes.value),
 		nodeNameToBeFocused: computed(() => nodeNameToBeFocused.value),
+		isMapperOpen: computed(() => isMapperOpen.value),
```

Comment:
Just to understand, is this to prevent the mutation of the value from outside? If so, nice, love that you took that into account 👌 Is there something like `ReadonlyRef` type that would achieve the same without the need for `computed` wrapping?